plugins/kubernetes/app/decorators/project_decorator.rb

```diff
@@ -54,6 +54,10 @@ def name_for_label
     name.parameterize('-')
   end
 
+  def deploy_watcher_actor_name
```

Comment:
Humm, I did this because I needed the same Actor name on the KuberDeployService, but you're right, I could have created a class method on the DeployWatcher that, given a project, returns the Actor name.

I'll change that. It was dumb to put this in the project.